drafts/2019-01-21-java-11.adoc

```diff
@@ -0,0 +1,293 @@
+
+[source,java]
+----
+// Pre-JDK 9
+List<String> list = Collections.unmodifiableList(Arrays.asList("one", "two", "three"));
+
+// JDK 9
+var list = List.of("one", "two", "three"); // List<String>
+var set =  Set.of("one", "two", "three");  // Set<String>
+var map =  Map.of("one", 1,                // Map<String,Integer>
+                  "two", 2,
+                  "three", 3);
+----
+
+== How to run Open Liberty on Java 11
+
+First of all, if you don't have a copy of it already, you will need to https://adoptopenjdk.net/index.html?variant=openjdk11&jvmVariant=openj9[download Java 11].
+
+NOTE: Currently Open Liberty supports Java 11.0.2 (or newer) on the OpenJ9 VM. Other Java VMs, such as Hotspot, are not supported due to some outstanding VM bugs.
+
+Next, you will need Open Liberty 19.0.0.1 or newer.
+
+If you are using https://www.openliberty.io/guides/maven-intro.html[Maven], here are the coordinates:
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>19.0.0.1</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for https://openliberty.io/guides/gradle-intro.html[Gradle]:
+[source,groovy]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[19.0.0.1,)'
+}
+----
+
+There are a number of ways to control the Java executable used by the Open Liberty server. The following list describes the order of precedence:
+
+. `JAVA_HOME` in `${server.config.dir}/server.env`
+. `JAVA_HOME` in `${wlp.install.dir}/etc/server.env`
+. `JAVA_HOME` set in the environment
+. searching the `PATH` in the environment
+
+To confirm that Open Liberty is using the intended Java level, you can check the first line of console output:
+
+----
+$ export JAVA_HOME=/path/to/java11
+
+$ export PATH=$JAVA_HOME/bin:$PATH
+
+$ bin/server run myServer
+
+Launching myServer (WebSphere Application Server 19.0.0.1/wlp-1.0.24.201901191232) on Eclipse OpenJ9 VM, version 11.0.2 (en_US)
+[AUDIT   ] CWWKE0001I: The server myServer has been launched.
+...
+----
+
+== Run a simple REST application on Java 11
+
+To run an application on Open Liberty that uses Java 11, we can start by defining the new compiler compliance level in the `build.gradle` file if using Gradle:
+
+[source,groovy]
+----
+sourceCompatibility = 11
+targetCompatibility = 11
+----
+
+Or, if you are using Maven, you can set the sourceCompatibility in your `pom.xml` like so:
+[source,xml]
+----
+<plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <version>3.8.0</version>
+    <configuration>
+        <release>11</release>
+    </configuration>
+</plugin>
+----
+
+NOTE: It is not required to set a compiler compliance level of Java 11 in order to run an existing application on Java 11, but in order to leverage any of the new language features you will need to do so.
```

Comment:
Are you not going to explain the compiler compliance level and how to set that?  I think it would be good to explain both java 8 and 11 compliance using the jdk 11 compiler.